docs/development/tool-comparison-gap-analysis.md

```
@@ -0,0 +1,510 @@
+| noexcept added/removed | `func_noexcept_added/removed` | Yes | No | |
+| Static↔non-static | `func_static_changed` | Yes | Yes | |
+| restrict qualifier change | `param_restrict_changed` | Yes | Yes | |
+| va_list transitions | `param_became/lost_va_list` | Yes | No | |
+| Pointer level change | `param_pointer_level_changed` | Yes | Partial | |
+
+### 3. Type / Struct Layout
+
+| Check | abicheck | abicc | libabigail | Notes |
+|-------|:--------:|:-----:|:----------:|-------|
+| Size changed | `type_size_changed` | Yes | Yes | |
+| Alignment changed | `type_alignment_changed` | Yes | Yes | Yes | |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 5; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)